### PR TITLE
[ADD] added website_cdn_support

### PR DESCRIPTION
**Pull request by Benniphx**
added the option for CDN support from the master for version 8

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Same comments about authorship than the other PR, and I think a simpler name but yet descriptive can be only _website_cdn_

File: website_cdn_support/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+CDN support for website
+=======================
+
+added the option for CDN support from the master for version 8
```
Comment (pedrobaeza) [inline website_cdn_support/README.md:4]: It can be good to explain what is CDN and for what can be used.

File: website_cdn_support/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+you can turn it on or of in the backend
```
Comment (pedrobaeza) [inline website_cdn_support/README.md:6]: Put it on the Configuration section with more details.

**Comment by rafaelbn** [comment]
Hi @Benniphx are you going to continue with this PR? Let me know please. If not are you agree to close it? Thanks!

**Comment by simahawk** [comment]
same as https://github.com/OCA/website/pull/71
feel free to reopen, thanks!

**Pull request closed without merging**